src/test.js

```diff
@@ -571,3 +571,28 @@ function skip( testName ) {
 
 	test.queue();
 }
+
+// Will be exposed as QUnit.only
+function only( testName, expected, callback, async ) {
+	var newTest;
+
+	if ( arguments.length === 2 ) {
+		callback = expected;
+		expected = null;
+	}
+
+	newTest = new Test({
+		testName: testName,
+		expected: expected,
+		async: async,
+		callback: callback
+	});
+
+	// First only wins, ignore subsequent onlys
+	if ( !QUnit.config.filter ) {
+		QUnit.config.queue.length = 0;
+		QUnit.config.filter = newTest.module.name + ": " + newTest.testName;
+	}
```

Comment:
Understood, but I'd still need to interrogate that state when adding additional tests. I can look up that variable on `QUnit.test` and `QUnit.skip` requests as well. The filter was effectively doing this for me previously.